### ISSUE TITLE
Response status code does not indicate success: 500 Internal Server error

### ISSUE THREAD

**Issue by neatcoder** [jfrog/setup-jfrog-cli#73]
Hello,

Our CIs started to fail yesterday when trying to restore nuget packages. Problem seems to be limited to Github actions only because we were able to restore packages on a local machine using JFrog CLI.

"Response status code does not indicate success: 500 (Internal Server Error)."

Is it related to the recent Github action update to 2.3.0?

**Versions**
- Setup JFrog CLI version: V2
- Workflow operating system: ubuntu-latest
- JFrog CLI version: latest
- Artifactory Version: 7.36.4

**Comment by yahavi** [comment]
@neatcoder,
Thanks for reporting this issue.
Setup JFrog CLI v2.3.0 contains the following improvements: https://github.com/jfrog/setup-jfrog-cli/releases/tag/v2.3.0
Also, we released yesterday JFrog CLI 2.15.1. Here are the release notes - https://github.com/jfrog/jfrog-cli/releases/tag/v2.15.1

Could you please try to run `jf rt ping` command? This will make sure there is a connection to Artifactory.
Also, you can try with JFrog CLI 2.15.0.

Please let me know about any new findings.

**Comment by neatcoder** [comment]
Tried with CLI 2.15.0:

```
Run jfrog/setup-jfrog-cli@v2.2.1
  with:
    version: 2.15.0
  env:
    SONAR_PROJECT_NAME: XXXX
    SONAR_HOSTNAME: https://sonar.dev.XXXX.com
    SWASHBUCKLE_VERSION: 6.2.[3](https://github.com/XXXX/XXXX/runs/6014237818?check_suite_focus=true#step:5:3)
    JFROG_CLI_BUILD_NAME: XXXX/XXXX
    JFROG_CLI_LOG_LEVEL: DEBUG
    DOTNET_ROOT: /home/runner/.dotnet
    JF_ARTIFACTORY_1: ***
Setup JFrog CLI
  Warning: The "JF_ARTIFACTORY_" prefix for environment variables is deprecated and is expected to be removed in v3. Please use the "JF_ENV_" prefix instead. The environment variables value should not be changed.
  /opt/hostedtoolcache/jf/2.15.0/x6[4](https://github.com/XXXX/XXXX/runs/6014237818?check_suite_focus=true#step:5:4)/jf c import ***
  20:14:47 [Info] Importing server ID 'bt'
  20:14:47 [Debug] Creating lock file:  /home/runner/.jfrog/locks/config/jfrog-cli.conf.lck.1847.1[6](https://github.com/XXXX/XXXX/runs/6014237818?check_suite_focus=true#step:5:6)4988088[7](https://github.com/XXXX/XXXX/runs/6014237818?check_suite_focus=true#step:5:7)914149[8](https://github.com/XXXX/XXXX/runs/6014237818?check_suite_focus=true#step:5:8)8[9](https://github.com/XXXX/XXXX/runs/6014237818?check_suite_focus=true#step:5:9)
  20:[14](https://github.com/XXXX/XXXX/runs/6014237818?check_suite_focus=true#step:5:14):47 [Debug] Releasing lock:  /home/runner/.jfrog/locks/config/jfrog-cli.conf.lck.1847.[16](https://github.com/XXXX/XXXX/runs/6014237818?check_suite_focus=true#step:5:16)49880887914149889
```

Here are the build logs:
```
  Run jfrog dotnet restore
  jfrog dotnet restore
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
  env:
    SONAR_PROJECT_NAME: XXXX
    SONAR_HOSTNAME: https://XXXXX.com
    SWASHBUCKLE_VERSION: 6.2.3
    JFROG_CLI_BUILD_NAME: XXXX/XXXXXX
    JFROG_CLI_LOG_LEVEL: DEBUG
    DOTNET_ROOT: /home/runner/.dotnet
    JFROG_CLI_ENV_EXCLUDE: *password*;*secret*;*key*;*token*;*auth*;JF_ARTIFACTORY_*;JF_ENV_*
    JFROG_CLI_OFFER_CONFIG: false
    CI: true
    JFROG_CLI_BUILD_NUMBER: 394[1](https://github.com/XXXX/XXXXX/runs/6014237818?check_suite_focus=true#step:8:1)
    JFROG_CLI_BUILD_URL: https://github.com/XXXX/XXXXX/actions/runs/[2](https://github.com/XXXX/XXXXX/runs/6014237818?check_suite_focus=true#step:8:2)16[3](https://github.com/XXXX/XXXXX/runs/6014237818?check_suite_focus=true#step:8:3)5[4](https://github.com/XXXX/XXXXX/runs/6014237818?check_suite_focus=true#step:8:4)0849
    JFROG_CLI_USER_AGENT: setup-jfrog-cli-github-action/2.2.1
20:14:47 [Debug] Found resolver in the config file /home/runner/work/XXXXX/XXXXX/src/.jfrog/projects/dotnet.yaml
20:14:48 [Debug] Usage Report: Sending info...
20:14:48 [Debug] Sending HTTP GET request to: https://XXXX.jfrog.io/artifactory/api/system/version
20:14:48 [Debug] using .NET SDK Core [5](https://github.com/XXXX/XXXXX/runs/6014237818?check_suite_focus=true#step:8:5).0.407

20:14:48 [Info] Running dotnet...
20:14:48 [Debug] Nuget config file created at: /tmp/jfrog.cli.temp.-1[6](https://github.com/XXXX/XXXXX/runs/6014237818?check_suite_focus=true#step:8:6)49880888-1536[7](https://github.com/XXXX/XXXXX/runs/6014237818?check_suite_focus=true#step:8:7)[8](https://github.com/XXXX/XXXXX/runs/6014237818?check_suite_focus=true#step:8:8)8011/jfrog.cli.nuget.40[9](https://github.com/XXXX/XXXXX/runs/6014237818?check_suite_focus=true#step:8:9)2017[10](https://github.com/XXXX/XXXXX/runs/6014237818?check_suite_focus=true#step:8:10)4
20:14:48 [Debug] Using access-token details for nuget authentication.
20:14:48 [Debug] The Artifactory version is: 7.36.4
20:14:48 [Debug] Sending HTTP POST request to: https://XXXX.jfrog.io/artifactory/api/system/usage
20:14:48 [Debug] Usage Report: Artifactory response: 200 
20:14:48 [Debug] Usage Report: Usage info sent successfully.
  Determining projects to restore...
/home/runner/.dotnet/sdk/5.0.407/NuGet.targets([13](https://github.com/XXXX/XXXXX/runs/6014237818?check_suite_focus=true#step:8:13)1,5): error : Response status code does not indicate success: 500 (). [/home/runner/work/XXXXX/XXXXX/src/XXXX.XXXXX.sln]
Error: 6 [Error] exit status 1
Error: Process completed with exit code 1.
```

**Comment by gitta-jfrog** [comment]
@neatcoder 

We found out this issue occurred as a result of releasing new versions for [librdkafka.redist](https://www.nuget.org/packages/librdkafka.redist/) Nuget package.

These versions include some naming conventions that Artifactory failed to handle. 
We fixed this issue as part of the [Artifactory v7.37.14 release.](https://www.jfrog.com/confluence/display/JFROG/Artifactory+Release+Notes#ArtifactoryReleaseNotes-Artifactory7.37.14)

If you are using an Artifactory Cloud instance, your instance should already be upgraded.

Please let me know if you managed to run your build successfully.
